Resources/doc/usage.rst

```diff
@@ -0,0 +1,180 @@
+Usage
+=====
+
+Add these two lines in your layout:
+
+.. code-block:: twig
+
+    <script src="{{ asset('bundles/fosjsrouting/js/router.js') }}"></script>
+    <script src="{{ path('fos_js_routing_js', {"callback": "fos.Router.setData"}) }}"></script>
+
+.. note::
+
+    If you are not using Twig, then it is no problem. What you need is to add
+    the two JavaScript files above loaded at some point in your web page.
+
+Generating URIs
+---------------
+
+It's as simple as calling:
+
+.. code-block:: javascript
+
+    Routing.generate('route_id', /* your params */)
```

Comment:
Symfony names it a route nam, not a route id. I suggest updating it to avoid confusion